oak/server/rust/oak_runtime/src/metrics.rs

```diff
@@ -0,0 +1,75 @@
+use hyper::{
+    header::CONTENT_TYPE,
+    service::{make_service_fn, service_fn},
+    Body, Request, Response, Server,
+};
+
+use log::info;
+use prometheus::{
+    labels, opts, register_counter, register_gauge, register_histogram_vec, Counter, Encoder,
+    Gauge, HistogramVec, TextEncoder,
+};
+use std::{convert::Infallible, net::SocketAddr};
+
+lazy_static::lazy_static! {
```

Comment:
Another approach would be to declare these counters as part of the `Runtime` itself (well,in a separate sub-object really) instead of global mutable variables. Feel free to go ahead with this if it makes things simpler, but worth keeping in mind how we expect things to look eventually.